README.md

```diff
@@ -1,15 +1,13 @@
 # Straitjacket
 
-An opinionated, one file framework for writing more reasonable code in a
-functional style which quarantines complexity.
+A one-file framework for reasonable code in a functional style.
 
 
 ## Introduction
 
-Software, like any creative endeavor, is controversial. "Good" is eternally
-subjective: what one likes, another detests. However, problems solved by code
-are logical. We may not agree on "good", but **we can objectively and
-dispassionately characterize code**. For example:
+Software is controversial. "Good" is subjective: what one likes, another hates.
```

Comment:
Maybe "software is controversial" should be "software design is controversial" or something like "The best way to design software is controversial"